client/components/editor/teaching-elements/Assessment/feedback/FeedbackItem.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <li>
+    <span class="answer-index">Answer {{ index + 1 }}:</span>
+    <template v-if="answer.length">{{ answer }}</template>
+    <i v-else>Answer not added.</i>
+    <quill-editor
+      v-if="isEditing"
+      :content="feedback"
+      :options="quillOptions"
+      @change="$emit('update', $event)">
+    </quill-editor>
+    <div v-else-if="feedback.length" v-html="feedback"></div>
+    <div v-else>
+      <i>Feedback not added.</i>
+    </div>
+  </li>
+</template>
+
+<script>
+import { quillEditor as QuillEditor } from 'vue-quill-editor';
+
+const QUILL_OPTIONS = {
+  modules: {
+    toolbar: [
+      ['bold', 'italic', 'underline', 'strike'],
+      ['blockquote', 'code-block'],
+      [{ list: 'ordered' }, { list: 'bullet' }],
+      [{ script: 'sub' }, { script: 'super' }],
+      [{ color: [] }, { background: [] }],
+      ['link']
+    ]
+  }
+};
+
+export default {
+  name: 'feedback-item',
+  props: {
+    answer: { type: String, default: '' },
+    feedback: { type: String, default: '' },
+    isEditing: { type: Boolean, required: true },
+    index: { type: Number, required: true }
+  },
+  computed: {
+    quillOptions() {
+      return QUILL_OPTIONS;
+    }
```

Comment:
You could further simplify it by using lambda:
```js
const quillOptions = { ... };

computed: {
  quillOptions: () => quillOptions
}
```

However it would be nice to make it overridable through prop. Add another prop called `options` or `config` and set this as default value?